.github/dependabot.yml

```diff
@@ -1,6 +1,20 @@
 version: 2
 updates:
   - package-ecosystem: "github-actions"
+    target-branch: trunk
```

Comment:
```suggestion
    target-branch: 6.6.x
```

Reply:
The second item uses 6.6.x, we need both in the same config if I understood it correctly